packages/jaeger-ui/src/utils/tracking/README.md

```diff
@@ -222,3 +222,19 @@ You get a lot for free when using Raven.js:
 - Some global handlers are added
 
 Implementing the above from scratch would require substantial effort. Meanwhile, Raven.js is well tested.
+
+### Steps to Verify the gaID Integration
+
+1.  **Setup**
```

Comment:
this is not a valid formatting for markdown, your bullets are not rendering correctly

Reply:
I suggest using single line, except for (3) which seems fine

Reply:
```suggestion
1. Setup: ...
```